doc/admin-guide/files/records.config.en.rst

```diff
@@ -2651,7 +2651,7 @@ Logging Configuration
    ===== ======================================================================
 
    For information on sending XML-based custom formats to the collation server,
```

Comment:
FWIW the main docs jira is [TS-4738](https://issues.apache.org/jira/browse/TS-4738).